src/index.js

```diff
@@ -0,0 +1,96 @@
+const state = {
+    tempVar : 76,
+    skyDisplay : {
+        'Sunny' : '☀️☀️☀️☀️☀️☀️☀️☀️☀️☀️',
+        'Cloudy' : '☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️',
+        'Rainy' : '🌧🌧🌧🌧🌧🌧🌧🌧🌧🌧',
+        'Stormy' : '⛈⛈⛈⛈⛈⛈⛈⛈⛈⛈',
+        'Snowy' : '🌨🌨🌨🌨🌨🌨🌨🌨🌨🌨',
+        'Gay' : '🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈',
+        'Meatball' : '🍝🍝🍝🍝🍝🍝🍝🍝🍝🍝'
+    },
+    landscapeDisplay : {
+        'Hot' : '🌵🌵🌵🌵🌵🌵🌵🌵🌵🌵',
+        'Warm' : '🌴🌴🌴🌴🌴🌴🌴🌴🌴🌴',
+        'Meh' : '🍂🍂🍂🍂🍂🍂🍂🍂🍂🍂',
+        'Cold' : '❄️❄️❄️❄️❄️❄️❄️❄️❄️❄️',
+        'Freezing' : '☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️'
+    },
+    cityName : ''
+};
+
+const upTemp = (event) => {
+    state.tempVar += 1;
+    const tempDisplay = document.getElementById("temp-display-big");
+    tempDisplay.textContent = `${state.tempVar} F`;
+    tempChecker(state.tempVar, tempDisplay);
+};
+
+const downTemp = (event) => {
+    state.tempVar -= 1;
+    const tempDisplay = document.getElementById("temp-display-big");
+    tempDisplay.textContent = `${state.tempVar} F`;
+    tempChecker(state.tempVar, tempDisplay);
+};
+
+const skyDisplayer = (event) => {
+    const skyDisplayElement = document.getElementById('sky-display');
+    const skyType = document.getElementById('sky-type');
+    const selection = skyType.value
+    skyDisplayElement.textContent = `${state.skyDisplay[selection]}`;
+    if (skyType.value === 'Gay') {
+        document.body.style.background = 'linear-gradient(45deg, red, orange, yellow, green, blue, indigo, violet, red)';
+    } else {
+        document.body.style.background = 'gray';
+    }
+};
+
+const tempChecker = (temp, tempDisplay) => {
+    const landscapeDisplay = document.getElementById('landscape-display');
+    if (temp > 80) {
+        tempDisplay.style.color = 'red';
+        landscapeDisplay.textContent = state.landscapeDisplay['Hot'];
+    } else if ((temp <= 80) && (temp > 60)) {
+        tempDisplay.style.color = 'orange';
+        landscapeDisplay.textContent = state.landscapeDisplay['Warm'];
+    } else if ((temp >= 45) && (temp <= 60)) {
+        tempDisplay.style.color = 'black';
+        landscapeDisplay.textContent = state.landscapeDisplay['Meh'];
+    } else if ((temp > 32) && (temp < 45)) {
+        tempDisplay.style.color = 'purple';
+        landscapeDisplay.textContent = state.landscapeDisplay['Cold'];
+    } else if (temp <= 32) {
+        tempDisplay.style.color = 'teal';
+        landscapeDisplay.textContent = state.landscapeDisplay['Freezing'];
+    }
```

Comment:
Consider the above logic and how similar all the branches are. We look for whether the temperature is within a range, and pick a color and landscape accordingly. What if we had a list of records that we could iterate through to find the values. As noted above, we really only need to store the bottom of each range, so we could set up something like
```js
  const tempRecords = [
    { lowerBound: 80, color: "red", landscape: "🌵🌵🌵🌵🌵🌵🌵🌵🌵🌵"},
    { lowerBound: 60, color: "other color", landscape: "other landscape"},
    ...
  ];
```

You have the beginnings of this already, but just need a little more data to maximize its utility.

With this data structure, we could simply iterate through the list and find the first record that has an lower bound _higher_ than our temperature, the use the prior record (the last record that had an lower bound _lower_ than our temperature) as the source of picking the color and landscape!